src/sinks/axiom.rs

```diff
@@ -41,20 +48,32 @@ pub struct AxiomConfig {
 
     /// The Axiom dataset to write to.
     #[configurable(metadata(docs::examples = "${AXIOM_DATASET}"))]
-    #[configurable(metadata(docs::examples = "vector.dev"))]
+    #[configurable(metadata(docs::examples = "vector_rocks"))]
     dataset: String,
 
     #[configurable(derived)]
     #[serde(default)]
     request: RequestConfig,
 
+    /// The compression algorithm to use.
+    ///
+    /// Supported values: `none` ( not recommended ), `zstd` ( recommended, default ), `gzip`, `deflate`
```

Comment:
```suggestion
```

In the docs the enum values are rendered (including the default) so I don't think we need this additional description.